ckanext/matomo/model.py

```diff
@@ -591,16 +586,16 @@ def get_all_visits_by_id(cls, resource_id):
         return visits
 
     @classmethod
-    def get_stat_counts_by_id_and_date_range(cls, resource_id,
-                                             start_date=datetime.today() - relativedelta(months=1),
-                                             end_date=datetime.today()):
-        total_visits = model.Session.query(func.sum(cls.visits)).filter(cls.resource_id == resource_id,
+    def get_stat_counts_by_id_and_date_range(cls, resource_id: str,
+                                             start_date: datetime = datetime.today() - relativedelta(months=1),
+                                             end_date: datetime = datetime.today()) -> Dict[str, int]:
```

Comment:
Default parameter values are the same for every call of the function. Use `None` as default value and generate the default values in the function body if they should not always point to the same value.

Reply:
Big oops 🙈